packages/neuron-ui/src/utils/ipc.ts

```diff
@@ -22,10 +22,10 @@ export const ipcRenderer = (() => {
 
 const getLiveCell = (outpoint: any) => ipcRenderer.send('getLiveCell', outpoint)
 const getCellsByTypeHash = (typeHash: string) => {
-  ipcRenderer.send(IPC_CHANNEL.GET_CELLS_BY_TYPE_HASH, typeHash)
+  ipcRenderer.send(IPCChannel.GetCellsByTypeHash, typeHash)
 }
 const sendCapacity = (addr: string, capacity: string) => {
```

Comment:
Can we use `address` instead of `addr`? Same for some objects that have `{ addr: ... }`.

Reply:
Of course, I will update it in another PR